juriscraper/AbstractSite.py

```diff
@@ -406,6 +408,12 @@ def _get_html_tree_by_url(self, url, parameters={}):
         tree.make_links_absolute(url)
         return tree
 
+    def _get_parsed_pdf(self, url):
+        """this is a stand in"""
+        pdf_bytes = requests.get(url, timeout=60).content
+        with pdfplumber.load(BytesIO(pdf_bytes)) as pdf:
+            return pdf
```

Comment:
This feels reminiscent of the problem we had getting JSON and doing something with it. Maybe whatever API we develop to handle that should also support PDFs?